libraries/AP_HAL_ChibiOS/CANFDIface.cpp

```diff
@@ -625,6 +700,17 @@ void CANIface::clear_rx()
 
 void CANIface::setupMessageRam()
 {
+#if defined(STM32G4)
+    const uint32_t base = SRAMCAN_BASE + 0x350 * can_interfaces[self_index_];
+    memset((void*)base, 0, 0x350);
+    MessageRam_.StandardFilterSA = base;
```

Comment:
the constants used below should be defined above, like we did for H7. https://github.com/ArduPilot/ardupilot/pull/16868/files#diff-1cf5017a76e0f1f33655884a6905eb86518ee0f85925d36704360a732bffb577R76

Reply:
fixed